packages/flutter_web_plugins/lib/src/navigation/url_strategy.dart

```diff
@@ -315,6 +313,6 @@ class BrowserPlatformLocation extends PlatformLocation {
   }
 
   @override
-  String getBaseHref() => getBaseElementHrefFromDom();
+  String? getBaseHref() => getBaseElementHrefFromDom();
   // String getBaseHref() => html.document.baseUri;
```

Comment:
remove?

Reply:
Oops, this was my bad. Please remove.

Reply:
Wow, good catch, I didn't even notice that comment!